### DISCUSSION TITLE
HCA: Apator Metra E-ITN40

### DISCUSSION THREAD

**Discussion by AlesVysocky** [wmbusmeters/wmbusmeters#1141] [Q&A]
Hello,
I would like to try to read my new HCA, but it is not supported and I am not sure where to start to try to decode the messages (if it is even possible). I don't have a knowledge in wm-bus, I only found this package, built it on Raspberry PI and bought im871A dongle.

I can see some messages which are not recognized and also some which can be decoded (I expect from neighborhood).
Not recognized messages are:
A5C20332446850425992506980A0119F2FF601D0025000C807C20A504D004D5812020200000000000000000000000021133E145F3D1926DAD5
A5C20332446850425992506980A0119F2FF601D0025000C807C20A504D004D5812020200000000000000000000000021133E145F3D1926DAD5
A5C20332446850425992506980A0119F2FF601D0025000C807C20A504D004D5812020200000000000000000000000021133E145F3D192753C4
A5C20332446850425992506980A0119F2FF601D0025000CA07C00A504D004D5812020200000000000000000000000021133E145F3D1927716B
A5C20332446850425992506980A0119F2FF601D0025000CA07C00A504D004D5812020200000000000000000000000021133E145F3D19256348
A5C20332446850425992506980A0119F2FF601D0025000CA07C20A504D004D5812020200000000000000000000000021133E145F3D192860D2

We have 57 devices in the building, I have IDs and can read the data with NFC, so there is a possible reverse engineering (if it is not encrypted) Even if there is NFC, I would like to try it with radio.

Some decoding info, but for older models and not in English: https://www.prodomy.cz/_files/ugd/e443c7_5c87990000ee40fda3b4cba113b13905.pdf
https://metra-su.cz/en/product/electronic-heat-cost-allocator-e-itn-40-with-integrated-radio-transmitter

Thank you

**Comment by AlesVysocky** [comment]
#1144  issue with a same device

**Comment by Alcazer** [comment]
> Hello, I would like to try to read my new HCA, but it is not supported and I am not sure where to start to try to decode the messages (if it is even possible). I don't have a knowledge in wm-bus, I only found this package, built it on Raspberry PI and bought im871A dongle.
> 
> I can see some messages which are not recognized and also some which can be decoded (I expect from neighborhood). Not recognized messages are: A5C20332446850425992506980A0119F2FF601D0025000C807C20A504D004D5812020200000000000000000000000021133E145F3D1926DAD5 A5C20332446850425992506980A0119F2FF601D0025000C807C20A504D004D5812020200000000000000000000000021133E145F3D1926DAD5 A5C20332446850425992506980A0119F2FF601D0025000C807C20A504D004D5812020200000000000000000000000021133E145F3D192753C4 A5C20332446850425992506980A0119F2FF601D0025000CA07C00A504D004D5812020200000000000000000000000021133E145F3D1927716B A5C20332446850425992506980A0119F2FF601D0025000CA07C00A504D004D5812020200000000000000000000000021133E145F3D19256348 A5C20332446850425992506980A0119F2FF601D0025000CA07C20A504D004D5812020200000000000000000000000021133E145F3D192860D2
> 
> We have 57 devices in the building, I have IDs and can read the data with NFC, so there is a possible reverse engineering (if it is not encrypted) Even if there is NFC, I would like to try it with radio.
> 
> Some decoding info, but for older models and not in English: https://www.prodomy.cz/_files/ugd/e443c7_5c87990000ee40fda3b4cba113b13905.pdf https://metra-su.cz/en/product/electronic-heat-cost-allocator-e-itn-40-with-integrated-radio-transmitter
> 
> Thank you

Hello, the data you sent belongs to a TCH (Techem) meter. Please try to obtain data for an Apator brand meter

https://wmbusmeters.org/analyze/32446850425992506980A0119F2FF601D0025000C807C20A504D004D58120202000000000000000000000000
